src/libdrakvuf/vmi.c

```diff
@@ -433,7 +439,14 @@ event_response_t pre_mem_cb(vmi_instance_t vmi, vmi_event_t* event)
             }
         }
         else
+        {
             event->slat_id = drakvuf->altp2m_idr;
+            if (event->mem_event.out_access & VMI_MEMACCESS_W)
+            {
+                PRINT_DEBUG("Somebody try to write to the shadow page, let's emulate it instead\n");
```

Comment:
Since `pass` is not being used in this case, it should be freed to solve the Travis error.